content/sensu-go/6.8/operations/maintain-sensu/disaster-recovery.md

```diff
@@ -0,0 +1,348 @@
+--etcd-initial-cluster-token sensu-backend-03 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-03:2380 \
+--etcd-advertise-client-urls http://sensu-backend-03:2379
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+3. Copy the etcd snapshot file to each cluster member so that all members will be restored using the same snapshot.
+
+4. Restore the Sensu configuration from the etcd snapshot to the running backend or cluster:
+
+   {{< language-toggle >}}
+
+{{< code shell "Restore a single backend" >}}
+ETCDCTL_API=3 etcdctl snapshot restore sensu_etcd_snapshot.db \
+--name sensu-backend-01 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-01 \
+--initial-advertise-peer-urls http://sensu-backend-01:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal 
+{{< /code >}}
+
+{{< code shell "Restore a clustered backend" >}}
+ETCDCTL_API=3 etcdctl snapshot restore sensu_etcd_snapshot.db \
+--name sensu-backend-01 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-01 \
+--initial-advertise-peer-urls http://sensu-backend-01:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal
+
+ETCDCTL_API=3 etcdctl snapshot restore snapshot.db \
+--name sensu-backend-02 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-02 \
+--initial-advertise-peer-urls http://sensu-backend-02:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal 
+
+ETCDCTL_API=3 etcdctl snapshot restore snapshot.db \
+--name sensu-backend-03 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-03 \
+--initial-advertise-peer-urls http://sensu-backend-03:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal 
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+You should see the restored Sensu configuration in the web UI or sensuctl output.
+
+## Best practices for backups
+
+The best backup plan depends on how much and how often your Sensu configuration changes, as well as your organization's disaster recovery and business continuity goals.
+Business needs will dictate the right plan for your Sensu installation, but following a few best practices helps ensure that backups are available and useful when you need them.
+
```

Comment:
Are there any other best practices to list here?

Is anything I have included inaccurate?

Reply:
I would say this is 👍